firmware/components/driver_sndmixer/sndmixer.c

```diff
@@ -235,7 +235,7 @@ static void sndmixer_task(void *arg) {
       int32_t s[2] = {0, 0};
       for (int ch = 0; ch < no_channels; ch++) {
         sndmixer_channel_t *chan = &channel[ch];
-        if (chan->source) {
+        if (chan->source  && (chan->flags & CHFL_PAUSED) == 0) {
```

Comment:
Nice catch. How about `!(chan->flags & CHFL_PAUSED)`?

Reply:
```suggestion
        if (chan->source  && !(chan->flags & CHFL_PAUSED)) {
```
Should work as well